tests/unit/algorithms/test_corrector.py

```diff
@@ -0,0 +1,1186 @@
+                        100,
+                        id=1,
+                        label=0,
+                        attributes={"truncated": False, "occluded": False},
+                    ),
+                    Bbox(
+                        0,
+                        0,
+                        100,
+                        100,
+                        id=2,
+                        label=1,
+                        attributes={"truncated": False, "occluded": False},
+                    ),
+                ],
+            ),
+        ],
+        categories={
+            AnnotationType.label: LabelCategories.from_iterable(
+                [
+                    ("a", "", ["truncated", "occluded"]),
+                    ("b", "", ["truncated", "occluded"]),
+                ]
+            ),
+        },
+    )
+
+
+@pytest.fixture
+def fxt_original_cls_dataset():
+    return Dataset.from_iterable(
+        [
+            DatasetItem(
+                id="normal_train_0",
+                subset="train",
+                annotations=[
+                    Label(id=0, label=0),
+                ],
+            ),
+            DatasetItem(
+                id="normal_train_1",
+                subset="train",
+                annotations=[
+                    Label(id=0, label=0),
+                ],
+            ),
+            DatasetItem(
+                id="normal_train_2",
+                subset="train",
+                annotations=[
+                    Label(id=0, label=1),
+                ],
+            ),
+            DatasetItem(
+                id="normal_train_3",
+                subset="train",
+                annotations=[
+                    Label(id=0, label=1),
+                ],
+            ),
+            DatasetItem(
+                id="undefined_label_0",
+                subset="train",
+                annotations=[
+                    Label(id=0, label=100),
+                ],
+            ),
+            DatasetItem(
+                id="multi_label_0",
+                subset="train",
+                annotations=[
+                    Label(id=0, label=0),
+                    Label(id=1, label=1),
+                ],
+            ),
+            DatasetItem(
+                id="few_sample_0",
+                subset="train",
+                annotations=[
+                    Label(id=0, label=3),
+                ],
+            ),
+            DatasetItem(
+                id="missing_label_0",
+                subset="train",
+            ),
+            DatasetItem(
+                id="normal_test_0",
+                subset="test",
+                annotations=[
+                    Label(id=0, label=0),
+                ],
+            ),
+        ],
+        categories={
+            AnnotationType.label: LabelCategories.from_iterable(
+                "label_" + str(label) for label in range(4)
+            ),
+        },
+    )
+
+
+@pytest.fixture
+def fxt_refined_cls_dataset():
```

Comment:
All the refined datasets can be created from their original dataset pairs. For example,
```python
def fxt_refined_cls_dataset(fxt_original_cls_dataset):
     dataset = deepcopy(fxt_original_cls_dataset)
     dataset.remove("multi_label_0")
```

Reply:
Yes, but we sometimes need to manipulate annotations and attributes per each item. Plus, if so, we may separate all tests from a `test_can_correct_data`.

Reply:
I updated. Thanks.